crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -11,25 +11,50 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_math::{prelude::*, FloatExt, FloatOrd};
 use bevy_picking::backend::prelude::*;
+use bevy_reflect::prelude::*;
 use bevy_render::prelude::*;
 use bevy_transform::prelude::*;
 use bevy_window::PrimaryWindow;
 
+/// Runtime settings for the [`SpriteBackend`].
+#[derive(Resource, Reflect)]
+#[reflect(Resource, Default)]
+pub struct SpriteBackendSettings {
+    /// When set to `true` picking will ignore any part of a sprite which is transparent
+    /// Off by default for backwards compatibility. This setting is provided to give you fine-grained
+    /// control over if transparency on sprites is ignored.
+    pub passthrough_transparency: bool,
+    /// How Opaque does part of a sprite need to be in order count as none-transparent (defaults to 10)
```

Comment:
Can you elaborate on the docs what the units are? 10 what? What is a valid range I could use, as a user?

Reply:
Can these two options be combined into a single enum? It doesn't make sense to have an alpha cutoff setting when alpha passthrough is disabled. e.g.

```rs
SpritePickingAlphaTest {
    /// Don't test alpha, only consider the rect of sprites.
    Ignore,
    /// Test the pixel alpha when running hit tests. Alpha values below this will not be considered for picking.
    Threshold(u8)
}
```